Dart/src/com/jetbrains/lang/dart/ide/errorTreeView/DartProblemsTableModel.java

```diff
@@ -367,6 +367,9 @@ public String getStatusText() {
         case Package:
           b.append("Filtered by current package");
           break;
+        case Directory:
+          b.append("Filtered by directory");
```

Comment:
Shouldn't it be 'Filtered by **current** directory', in compliance with the radio button label?

BTW do you know preferences about using 'directory' vs 'folder' in the IDE UI?